ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -657,6 +657,40 @@ public async Task EditMission(int missionNo)
                 }
             }
         }
+		
+        [Command("edytuj-misje")]
+        [Summary("Po podaniu nazwy misji jako parametru włączy edycje danej misji (część bez zespołów).")]
+        [ContextDMOrChannel]
+        public async Task EditMission([Remainder] string title)
+        {
+            var currentlyEditedMission = SignupsData.GetCurrentlyEditedMission(Context.User.Id);
+
+            if (currentlyEditedMission == null)
+            {
+                var missionToBeEdited = SignupsData.Missions.FirstOrDefault(x => x.Title == title);
```

Comment:
It doesn't always work if you call the command `AF!edytuj-misje mission-title`. The channel name could be `#mission-title` but the mission name is e.g. `Mission Title` and it might be confusing so the comparison should be done some other way if we allow to specify the string instead of channel "mention".

Reply:
I think you should reuse the channel name creation from mission title. Get the mission title, create a channel name for it and then compare the strings. When using the `IGuildChannel` like suggested above, these 2 should always equal.

Reply:
But why? The descriptions states clearly that the argument is mission name. Not channel name or anything else. Making this comparison case insensitive is the only thing I see reasonable.  User writes a name of the mission like it is everywhere except channel name so why using channel name?

Reply:
What's the point of copy-pasting whole mission title (because you will need to enter the mission channel, copy the title and come back to input a command)? Why not just use the `IGuildChannel` and use the channel mention which is the only intuitive way of invoking commands like this? Just like for `AF!ban` you would need to enter the user name instead of just mentioning him.

Reply:
OK. Now I get it and it makes sense. I thought about writing channel name by hand.